src/Components/DrillIllustration.js

```diff
@@ -8,10 +8,69 @@ import { IllustrationType } from '../Fixtures';
 import theme from '../styles/theme.style';
 import iconRedo from '../../assets/redo_arrow.png';
 import buttonValidation from '../../assets/button_validation_ultra_light.png';
+import buttonValidationGradient from '../../assets/button_validation_gradient.png';
+import { Easing } from 'react-native-reanimated';
 
 const DrillIllustration = props => {
   const [currentStepIndex, setStepIndex] = useState(0);
 
+  const opacityUnchecked = useRef(new Animated.Value(1)).current;
+  const opacityChecked = useRef(new Animated.Value(0)).current;
```

Comment:
I'd recommend using `opacityChecked = opacityUnchecked.interpolate({inputRange: [0, 1],
  outputRange: [1, 0]})`, this way you don't need to Animated.timing both values.